**References**

Felsenstein, J. (1973). Maximum likelihood and minimum-steps methods for estimating evolutionary trees from data on discrete characters. Systematic Biology, 22(3), 240–249. https://doi.org/10.1093/sysbio/22.3.240

Graham, R., Knuth, D., & Patashnik, O. (1994). Concrete Mathematics. Addison-Wesley Longman Publishing Co., Inc.75 Arlington Street, Suite 300 Boston, MAUnited States.

Katoh, K., & Standley, D. M. (2013). MAFFT Multiple Sequence Alignment Software Version 7: Improvements in Performance and Usability. Molecular Biology and Evolution, 30(4), 772–780.

Minh, B. Q., Schmidt, H. A., Chernomor, O., Schrempf, D., Woodhams, M. D., Von Haeseler, A., Lanfear, R., & Teeling, E. (2020). IQ-TREE 2: New Models and Efficient Methods for Phylogenetic Inference in the Genomic Era. Molecular Biology and Evolution, 37(5), 1530–1534. https://doi.org/10.1093/molbev/msaa015

Nascimento, F. F., dos Reis, M., & Yang, Z. H. (2017). A biologist’s guide to Bayesian phylogenetic analysis. Nature Ecology & Evolution, 1(10), 1446–1454. https://doi.org/10.1038/s41559-017-0280-x

Negative binomial distribution. (2023). https://en.wikipedia.org/wiki/Negative_binomial_distribution

Suyama, M., Torrents, D., & Bork, P. (2006). PAL2NAL: Robust conversion of protein sequence alignments into the corresponding codon alignments. Nucleic Acids Research. https://doi.org/10.1093/nar/gkl315

Yang, Z. (2006). Computational Molecular Evolution. Oxford University Press. http://abacus.gene.ucl.ac.uk/CME/

Yang, Z. (2014). Molecular Evolution: A Statistical Approach. Oxford University Press. http://abacus.gene.ucl.ac.uk/MESA/
